lib/cfg-grammar.y

```diff
@@ -466,6 +468,12 @@ DNSCacheOptions *last_dns_cache_options;
 %type   <num> positive_integer64
 %type   <num> nonnegative_integer
 %type   <num> nonnegative_integer64
+%type	<cptr> path_inactive
```

Comment:
Do we need this level of granularity?

I am thinking of sustainability effort. When someone creates a new driver with a file option, or just adds a new file option to an existing driver, it would take some effort just to classify which of these types to use.

`path_secret` sounds reasonable: we might want to handle such files differently in the debun.
`existing_path` also sounds handy: as a very early assert if an entity in the config exists or not. (I would just call it `path` though).

For the others, I am not sure. These classifications induce some confusion and complexity on the code, and we need to make sure the win matches or larger than the cost.

Reply:
@furiel : Yes, you have a good point. I am removing some path types.
I am keeping only 3 types as of now.
```
1. path - Check for an existing path
2. path_inactive - No check for an existing path
3. path_sercret - for key files
```
Let me know if this is good enough for now or not ?

Reply:
Sounds much simpler, thanks.

Sorry for nitpicking, but lets call `path_no_check` instead of `path_inactive`. It is more expressive.

Reply:
@furiel : Okay, I am using `path_no_check` now